src/P3Scheme.jl

```diff
@@ -246,12 +301,12 @@ function DSD_N₀(p3::PSP3, N::FT, λ::FT) where {FT}
 end
 
 """
-    q_(p3, ρ, F_r, N_0, λ, μ, D_min, D_max)
+    q_(p3, ρ, F_r, λ, μ, D_min, D_max)
 
  - p3 - a struct with P3 scheme parameters
  - ρ - bulk ice density (ρ_i for small ice, ρ_g for graupel) [kg/m^3]
  - F_r - rime mass fraction [q_rim/q_i]
- - N_0 - intercept parameter of N′ gamma distribution
```

Comment:
This now also returns q/N and not mass density? We should update the docstring below